docs/container_registry.md

```diff
@@ -103,103 +120,17 @@ Then update the `volumes` and `environment` sections of your `gitlab` container:
-    return 301 https://$http_host:$request_uri;
-}
-```
-
 # Configuration
 
 ## Available Parameters
 
-Here is an example of all configuration parameters that can be used in the GitLab container.
-
-```yml
-...
-gitlab:
-    ...
-    environment:
-    - GITLAB_REGISTRY_ENABLED=true
-    - GITLAB_REGISTRY_HOST=registry.gitlab.example.com
-    - GITLAB_REGISTRY_API_URL=http://registry:5000
-    - GITLAB_REGISTRY_KEY_PATH=/certs/registry-auth.key
-    - GITLAB_REGISTRY_ISSUER=gitlab-issuer
-    - SSL_REGISTRY_KEY_PATH=/certs/registry.key
-    - SSL_REGISTRY_CERT_PATH=/certs/registry.crt
-```
-
-where:
-
-| Parameter | Description |
-| --------- | ----------- |
-| `GITLAB_REGISTRY_ENABLED ` | `true` or `false`. Enables the Registry in GitLab. By default this is `false`. |
-| `GITLAB_REGISTRY_HOST `    | The host URL under which the Registry will run and the users will be able to use. |
-| `GITLAB_REGISTRY_PORT `    | The port under which the external Registry domain will listen on. |
-| `GITLAB_REGISTRY_API_URL ` | The internal API URL under which the Registry is exposed to. |
-| `GITLAB_REGISTRY_KEY_PATH `| The private key location that is a pair of Registry's `rootcertbundle`. Read the [token auth configuration documentation][token-config]. |
-| `GITLAB_REGISTRY_PATH `    | This should be the same directory like specified in Registry's `rootdirectory`. Read the [storage configuration documentation][storage-config]. This path needs to be readable by the GitLab user, the web-server user and the Registry user *if you use filesystem as storage configuration*. Read more in [#container-registry-storage-path](#container-registry-storage-path). |
-| `GITLAB_REGISTRY_ISSUER`  | This should be the same value as configured in Registry's `issuer`. Otherwise the authentication will not work. For more info read the [token auth configuration documentation][token-config]. |
-| `SSL_REGISTRY_KEY_PATH `    | The private key of the `SSL_REGISTRY_CERT_PATH`. This will be later used in nginx to proxy your registry via https. |
-| `SSL_REGISTRY_CERT_PATH `    | The certificate for the private key of `SSL_REGISTRY_KEY_PATH`. This will be later used in nginx to proxy your registry via https. |
-
-For more info look at [Available Configuration Parameters](https://github.com/sameersbn/docker-gitlab#available-configuration-parameters).
-
-A minimum set of these parameters are required to use the GitLab Container Registry feature.
-
-```yml
-...
-gitlab:
-    environment:
-    - GITLAB_REGISTRY_ENABLED=true
-    - GITLAB_REGISTRY_HOST=registry.gitlab.example.com
-    - GITLAB_REGISTRY_API_URL=http://registry:5000
-    - GITLAB_REGISTRY_KEY_PATH=/certs/registry-auth.key
-    - GITLAB_REGISTRY_ISSUER=gitlab-issuer
-...
-```
+Here is an example of all configuration parameters that can be used in the GitLab container. See the [Available Configuration Parameters](https://github.com/sameersbn/docker-gitlab#available-configuration-parameters).
```

Comment:
If you remove those, then we should also remove the sentence "Here is an example of all configuration parameters ...". They are no longer here. But I agree that those could probably be removed.